codebase/superdarn/src.bin/tk/tool/make_raw.1.8/make_raw.c

```diff
@@ -214,11 +219,27 @@ int main (int argc,char *argv[]) {
     exit(-1);
   }
 
+  command[0]=0;
+  n=0;
+  for (c=0;c<argc;c++) {
+    n+=strlen(argv[c])+1;
+    if (n>127) break;
```

Comment:
```suggestion
    // if the commannd is too long just cut it off
    n+=strlen(argv[c])+1;
    if (n>127) break;
```
Should we make a warning? something to think about the future.